### PR TITLE
Improvement/opening git dialog

### PR DESCRIPTION
**Pull request by justinehell**
Factorize opening of git dialogs

### CODE REVIEW

File: src/components/menu/GitBranchMenu.vue [hunk 1]
```diff
@@ -234,47 +234,16 @@ async function initBranches() {
 }
 
 /**
- * Send event to open the GitNewBranchDialog.
+ * Send event to open the dialog corresponding to the key and close the menu..
```
Comment (k-y0u) [inline src/components/menu/GitBranchMenu.vue:237]: remove one of the dote at the ends of the line

File: tests/unit/components/menu/GitBranchMenu.spec.js [hunk 2]
```diff
@@ -129,44 +126,15 @@ describe('Test component: GitBranchMenu', () => {
...
-        wrapper.vm.openGitCommitDialog();
-        expect(DialogEvent.next).toBeCalled();
-        expect(wrapper.vm.menu.hide).toBeCalled();
-      });
-    });
-
-    describe('Test function: openLog', () => {
-      it('should call dialog event and hide menu', () => {
+    describe('Test function: openDialog', () => {
+      it('should call dialog event with "GitStatus" and hide menu', () => {
```
Comment (k-y0u) [inline tests/unit/components/menu/GitBranchMenu.spec.js:130]: should call dialog event with the given key and hide menu

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: tests/unit/components/menu/GitBranchMenu.spec.js (hunk 2 above)
Comment (justinehell) [reply tests/unit/components/menu/GitBranchMenu.spec.js]: done

File: src/components/menu/GitBranchMenu.vue (hunk 1 above)
Comment (justinehell) [reply src/components/menu/GitBranchMenu.vue]: done

**Review by k-y0u — APPROVED** [review]

**Pull request merged**